src/main/java/com/google/codeu/servlets/MarkerServlet.java

```diff
@@ -68,6 +87,8 @@ public void storeMarker(Marker marker) {
     markerEntity.setProperty("lat", marker.getLat());
     markerEntity.setProperty("lng", marker.getLng());
     markerEntity.setProperty("content", marker.getContent());
+    markerEntity.setProperty("userName", marker.getUserName());
+    markerEntity.setProperty("skillLevel", marker.getSkillLevel());
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(markerEntity);
```

Comment:
i would create a new function

```
@Override
public void doDelete(HttpServletRequest request, HttpServletResponse response) {
  datastore.delete("Marker");
}
```

i would also move the storeMarker code to datastore.java and invoke it with datstore.storeMarker() 
https://github.com/HumbertoGlez/CodeU-Team25/blob/master/src/main/java/com/google/codeu/data/Datastore.java